public/api/conf/2.0/schemas/wco/declaration/WCO_DS/WCO_DEC_DS_2_DMS.xsd

```diff
@@ -1093,7 +1103,7 @@
     <xs:simpleContent>
       <xs:restriction base="udt:CodeType">
         <xs:maxLength value="17"/>
-        <xs:pattern value=".*[^\s].*"/>
+        <xs:pattern value="^(\?!INV).*"/>
```

Comment:
this was left here by mistake, correct value is `<xs:pattern value=".*[^\s].*"/>`